spec/02-integration/05-proxy/04-ssl_spec.lua

```diff
@@ -56,6 +63,19 @@ describe("SSL", function()
     helpers.stop_kong()
   end)
 
+  describe("global SSL", function()
+    it("fallbakcs on the global SSL certificate", function()
```

Comment:
- typo: fallbacks
- rename to "fallbacks on the default proxy SSL certificate when SNI is not provided by client"